adminSiteClient/MultiDimIndexPage.tsx

```diff
@@ -86,21 +87,31 @@ function createColumns(
             render: (_, record) => {
                 return (
                     <a
-                        href={urljoin(
-                            BAKED_BASE_URL,
-                            `/admin/grapher/${record.slug}`
-                        )}
+                        href={
+                            record.catalogPath
+                                ? urljoin(
+                                      BAKED_BASE_URL,
```

Comment:
This will not work on live, where `BAKED_BASE_URL` is ourworldindata.org but `ADMIN_BASE_URL` is admin.owid.io.

Reply:
Thanks, fixing!